src/containers/Artist/ArtistPerformances/ArtistPerformances.js

```diff
@@ -1,14 +1,32 @@
 import React, { Component } from 'react'
+import ReactPaginate from 'react-paginate'
 import ArtistPerformance from '../../../components/Artist/ArtistPerformance/ArtistPerformance'
 
 class ArtistPerformances extends Component {
+  handlePageClick() {
+    console.log('The page has changed!')
+  }
```

Comment:
It's a stub for future functionality that'll likely to happen as soon as we start receiving data and see how many of listed items we want / should show on our pages.

Reply:
Probably OK. I just wanted to understand why it's there. And ideally have some idea of when the work that's going to motivate having a real implementation here might get done -- the longer a dummy implementation sits, the higher the chances for rot.

Reply:
Makes sense, I'll remove it by now and since the documentation of the library is quite informative when actual page change will happen we can get back to the implementation of the function!